modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -459,4 +459,100 @@ definition {
 		}
 	}
 
+	@description = "LPS-173894 - Verify that it's possible to send an e-mail notification via an Action (Delete)"
+	@priority = 5
+	test CanSendEmailViaActionDelete {
+		property custom.properties = "feature.flag.LPS-173894=true";
+
+		task ("Given: an Email Notification Template") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "Email Body",
+				entryTo = "test@liferay.com",
+				fromAddress = "Liferay",
+				fromName = "Test Test",
+				name = "Notification Template",
+				notificationType = "Email",
+				subject = "Subject");
+
+			PortletEntry.save();
+		}
+
+		task ("And: an Custom Object with an Action (Delete Entry)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 161916",
+				objectName = "CustomObject161916",
+				pluralLabelName = "Custom Objects 161916");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "true",
+				objectName = "CustomObject161916");
```

Comment:
Use LPS from the ticket to create this Custom Object. Remember that change for all Objects.